src/ser.rs

```diff
@@ -1,6 +1,11 @@
 // Copyright (c) The Diem Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
+// `Error::other` doesn't exist on the MSRV.
+#![allow(clippy::io_other_error)]
+// And neither does `clippy::io_other_error`.
+#![allow(unknown-lints)]
```

Comment:
This doesn't work. Should we just bump the MSRV?

Reply:
Oh I think I just misnamed the lint.  Fixed in [a2260bf](https://github.com/zefchain/bcs/pull/9/commits/a2260bfd5efef521c03b05eae2a86d572fa5d02a).

We could, but in order to get [`std::io::Error::other`](https://doc.rust-lang.org/std/io/struct.Error.html#method.other) we'd need to bump to 1.74, which is a pretty major bump.  I'm okay with it if yous are, but it's not a decision I'd take by myself.